renderer.js

```diff
@@ -1013,38 +1019,60 @@ class SDFGRenderer {
         this.toolbar.appendChild(d);
 
         // Enter object moving mode
-        d = document.createElement('button');
-        d.className = 'button';
-        d.innerHTML = '<i class="material-icons">open_with</i>';
-        d.style = 'padding-bottom: 0px; user-select: none';
-        d.onclick = () => {
+        let move_mode_btn = document.createElement('button');
+        move_mode_btn.className = 'button';
+        move_mode_btn.innerHTML = '<i class="material-icons">open_with</i>';
+        move_mode_btn.style = 'padding-bottom: 0px; user-select: none';
+        move_mode_btn.onclick = () => {
             this.move_mode = !this.move_mode;
             if (this.move_mode) {
-                d.innerHTML = '<i class="material-icons">done</i>';
-                d.title = 'Exit object moving mode';
+                move_mode_btn.innerHTML = '<i class="material-icons">done</i>';
+                move_mode_btn.title = 'Exit object moving mode';
             } else {
-                d.innerHTML = '<i class="material-icons">open_with</i>';
-                d.title = 'Enter object moving mode';
+                move_mode_btn.innerHTML =
+                    '<i class="material-icons">open_with</i>';
+                move_mode_btn.title = 'Enter object moving mode';
             }
         };
-        d.title = 'Enter object moving mode';
-        this.toolbar.appendChild(d);
+        move_mode_btn.title = 'Enter object moving mode';
+        this.toolbar.appendChild(move_mode_btn);
+
+        // Enter box selection mode
+        let box_select_btn = document.createElement('button');
+        box_select_btn.className = 'button';
+        box_select_btn.innerHTML =
+            '<i class="material-icons">select_all</i>';
```

Comment:
what about `highlight_alt`? https://material.io/resources/icons/?icon=highlight_alt&style=baseline

Reply:
Unfortunately doesn't work, a temporary alternative was added for now.